packages/react/src/Form/Text/TextInput.stories.tsx

```diff
@@ -0,0 +1,147 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { MessageTypes } from "../core";
+import { TextInput } from "./TextInput";
+import { TYPES } from "./constants";
+
+const meta: Meta<typeof TextInput> = {
+  component: TextInput,
+  title: "Components/Form/Input/Text",
+  argTypes: { onChange: { action: "onChange" } },
+};
+
+export default meta;
+
+type Story = StoryObj<React.ComponentProps<typeof TextInput>>;
```

Comment:
Il me semble que ca suffit cette syntaxe :

```suggestion
type Story = StoryObj<typeof TextInput>;
```

Reply:
En effet, c'est utilisé ici https://github.com/AxaFrance/react-toolkit/blob/master/packages/Form/Input/text/README.md?plain=1#L246
Je vais remplacer